.jenkins/pytorch/test.sh

```diff
@@ -284,6 +284,18 @@ test_inductor_huggingface() {
   test_inductor_benchmark huggingface
 }
 
+test_inductor_huggingface_perf() {
```

Comment:
Some refactoring has been done upstream, https://github.com/pytorch/pytorch/blob/17941b12e0f0aeb018f5bb68502eca78bdf6f60b/.jenkins/pytorch/test.sh#L259. Please update correspondingly.

Reply:
Done, thanks for the reminder.